src/Controls/tests/TestCases.HostApp/Issues/Issue21488.cs

```diff
@@ -0,0 +1,27 @@
+namespace Maui.Controls.Sample.Issues;
+
+[XamlCompilation(XamlCompilationOptions.Compile)]
+[Issue(IssueTracker.Github, "21488", "Button text doesn't update when CharacterSpacing is applied", PlatformAffected.iOS)]
+public partial class Issue21488 : ContentPage
+{
+	public Issue21488()
+	{
+		var buttonWithoutCharacterSpacing = new Button();
+		var buttonWithCharacterSpacing = new Button() { CharacterSpacing = 15 };
```

Comment:
Could include another Button using the ContentLayout property (an icon at the left for example)?